<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор справочников</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="driversTab">
       <attribute name="title">
        <string>Водители</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="1">
         <widget class="QComboBox" name="driverSuppliers"/>
        </item>
        <item row="10" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="driverPhone">
          <property name="inputMask">
           <string>+7(999)999-99-99</string>
          </property>
          <property name="placeholderText">
           <string>Номер телефона</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="driverEmail">
          <property name="placeholderText">
           <string>Адрес эл. почты (e-mail)</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QLineEdit" name="driverComment">
          <property name="placeholderText">
           <string>Комментарий</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableView" name="driversTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="driverTruck"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="driverName">
          <property name="placeholderText">
           <string>Фамилия Имя Отчество</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="driverSave">
          <property name="text">
           <string>Записать</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="7">
         <widget class="QLabel" name="driverPhoto">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="addPhoto">
          <property name="text">
           <string>Загрузить фото</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>driverName</tabstop>
  <tabstop>driverPhone</tabstop>
  <tabstop>driverEmail</tabstop>
  <tabstop>driverTruck</tabstop>
  <tabstop>driverSuppliers</tabstop>
  <tabstop>driverComment</tabstop>
  <tabstop>driverSave</tabstop>
  <tabstop>addPhoto</tabstop>
  <tabstop>driversTable</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
